palisade-service/mvn_dependency_tree.txt

```diff
@@ -136,13 +136,13 @@ io.etcd/jetcd-launcher/version = 0.3.0
 
 io.etcd/jetcd-resolver/version = 0.3.0
 
-io.fabric8/kubernetes-client/version = 4.3.0
+io.fabric8/kubernetes-client/version = 4.2.0
 
 io.fabric8/kubernetes-model/version = 4.2.0
 
 io.fabric8/kubernetes-model-common/version = 4.2.0
 
-io.fabric8/kubernetes-server-mock/version = 4.3.0
+io.fabric8/kubernetes-server-mock/version = 4.2.0
 
```

Comment:
Looks like we have gone backwards with the kubernetes-server-mock version - can you have a look why this is please?

Reply:
Amended Kubernetes client and mock versions so that they are consistent across all poms.